modules/claims_api/lib/bd/bd.rb

```diff
@@ -33,23 +34,24 @@ def search(claim_id, file_number)
     # Upload document of mapped claim
     #
     # @return success or failure
-    def upload(claim, pdf_path, file_number)
-      body = generate_upload_body(claim, pdf_path, file_number)
+    def upload(claim:, pdf_path:, file_number: nil)
+      @multipart = true
```

Comment:
you can set multipart in the initialize.....

Reply:
True, but if the `upload` endpoint has to be multipart, I'd feel like it's easier for a dev consuming the BD lib to not have to remember to set it to true when using the `upload` method.